tests/util/test_list.cpp

```diff
@@ -35,10 +35,10 @@ int *get_int_data(list_node_t *lnode) {
     return (int *)get_data(lnode);
 }
 
-bool iterate_int(list_node_t *lnode)
+BOOL iterate_int(list_node_t *lnode)
```

Comment:
Prefer use of standard library type to custom type.

Reply:
BOOL will work with older compilers. Leave the requirement of C11 compliancy for the next version.